src/pages/search/advanced_search/NviReportedYearFilter.tsx

```diff
@@ -0,0 +1,45 @@
+import { MenuItem, TextField } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useRegistrationsQueryParams } from '../../../utils/hooks/useRegistrationSearchParams';
+import { getNviYearFilterValues } from '../../../utils/nviHelpers';
+import { syncParamsWithSearchFields } from '../../../utils/searchHelpers';
+
+const relevantNviYears = getNviYearFilterValues(new Date().getFullYear() - 1);
+
+export const NviReportedYearFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const { scientificReportPeriodSince, scientificIndexStatus } = useRegistrationsQueryParams();
+
+  return (
+    <TextField
+      data-testid={dataTestId.startPage.advancedSearch.nviReportedYearField}
+      select
+      disabled={!scientificIndexStatus}
+      value={scientificReportPeriodSince ?? ''}
+      size="small"
+      label={t('search.advanced_search.nvi_reported_year')}
+      sx={{ width: '10rem' }}
+      onChange={(event) => {
+        const selectedYear = event.target.value;
+        const syncedParams = syncParamsWithSearchFields(searchParams);
+        if (selectedYear) {
+          syncedParams.set(ResultParam.ScientificReportPeriodBeforeParam, (+selectedYear + 1).toString());
+          syncedParams.set(ResultParam.ScientificReportPeriodSinceParam, selectedYear);
```

Comment:
Kanskje litt uheldig at vi trikser til dette med å sette begge, i stedet for å samle det til en `ScientificReportPeriodYear` elns, så splitte opp til 2 i selve API-kallet. Tror det hadde forenklet en god del, men må i så fall justere noen andre plasser som gjør dette slik i dag også da